packages/frontend/amp/lib/tag-utils.ts

```diff
@@ -0,0 +1,22 @@
+export const filterForTagsOfType = (
+    tags: TagType[],
+    tagType: string,
+): TagType[] => {
+    return tags.filter(
+        tag =>
+            tag.type === tagType ||
+            (tag.type === 'PaidContent' && tag.paidContentType === tagType),
+    );
+};
+
+export type Tone = 'isOpinion' | 'isPaidContent' | 'isDefault';
```

Comment:
This is my fault for using Opinion everywhere 🤦‍♂️ but perhaps we should use the actual tone values here (i.e. exactly what the tags contain) - 'comment', 'advertisement-features', '...' ? The benefit is that developers will find it easier to understand the connection between DCR and CAPI/Frontend modelling.

Reply:
The `is` prefix is probably also worth dropping as types tend to be nouns rather than predicates if that makes sense.

Reply:
Fixed this to match CAPI